db/migrate/20231012080842_add_browses_count_to_vacancies.rb

```diff
@@ -0,0 +1,5 @@
+class AddBrowsesCountToVacancies < ActiveRecord::Migration[7.0]
+  def change
+    add_column :vacancies, :browses_count, :integer
```

Comment:
default value 0, this filed also need to show in annotate in vacancy model class and factory